manifests/params.pp

```diff
@@ -229,7 +229,7 @@
       'intercept_form_submit' => 'mod_intercept_form_submit',
       'ldap'                  => $::apache::version::distrelease ? {
         '7'     => 'mod_ldap',
-        default => undef,
+        default => 'mod_ldap',
```

Comment:
This makes the selector useless (all path lead to `'ldap' => 'mod_ldap'`), it should be removed then.

Reply:
Yes, you are right